clients/admin-ui/src/features/privacy-requests/types.ts

```diff
@@ -112,27 +112,27 @@ export type RetryRequests = {
 };
 
 export interface MessagingConfigResponse {
-  fides: {
-    storage: {
-      active_default_storage_type: string;
-    };
+  storage: {
+    active_default_storage_type: string;
   };
 }
 
```

Comment:
These interfaces can be deleted after you have generated the API models request/response.

Reply:
You might need to show me what you mean by this.